public/js/views/management/pay-methods.jsx

```diff
@@ -20,6 +21,10 @@ export default class PayMethods extends Component {
 
   handleDelPayMethod = e => {
     e.preventDefault();
+    if (isDisabled(e.target)) {
```

Comment:
Yeah I know what you mean. We'll have to be thorough! We need a click middleware.

I'd considered using pointer-events in CSS - I could add that too (as a saftety net) . But support isn't good enough for our needs http://caniuse.com/#feat=pointer-events - hence I thought we should just deal with it in the click. The other consideration was if we use pointer-events that would mask the problem in IE 9/10 because it would be fixed in everything else just not there.

Another possibility would be to replace the link with a button here since we're doing click events anyway - though that feels a little wrong.

I'll go ahead with this for now but let's add this to the to-discuss list (maybe before or after the standup).